src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -61,6 +68,22 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                 index = -1;
             }
         }
+        const blackBoxComment =
+            vulnerable.moreVotesToWin > vulnerableVotes.moreVotesToWin
+                ? [
+                      " hadde nærmest kvotient. ",
+                      <span key="99" style={{ color: "orange" }}>
```

Comment:
This is a "hack" to allow it to pass without warnings due to key..?

Reply:
Yep.

```
<span key={vulnerableVotes.partyCode} className="has-text-warning">
    {showPartyName[vulnerableVotes.partyCode]}
</span>
```

Legit if the variable is presented only once?